xc/xc7/fasm2bels/make_routes.py

```diff
@@ -102,7 +102,7 @@ def output_builder(fixed_route):
 
     # TCL cannot express 1-length list, so add an additional element to
     # prevent TCL from collapsing the 1-length list.
```

Comment:
The above comment is important and needs to be handled.  I'll make a comment about it in a minute.